diff --git a/node_modules/metro-config/src/loadConfig.js b/node_modules/metro-config/src/loadConfig.js
index 7ac9d88..1b66a69 100644
--- a/node_modules/metro-config/src/loadConfig.js
+++ b/node_modules/metro-config/src/loadConfig.js
@@ -289,7 +289,13 @@ async function loadConfigFile(absolutePath) {
       }
     } catch (e) {
       try {
-        const configModule = await import(absolutePath);
+        // Fix for Windows ESM loader
+        const url = require('url');
+        const importPath = process.platform === 'win32' 
+          ? url.pathToFileURL(absolutePath).href 
+          : absolutePath;
+          
+        const configModule = await import(importPath);
         config = await configModule.default;
       } catch (error) {
         let prefix = `Error loading Metro config at: ${absolutePath}\n`;
